app/grants/views.py

```diff
@@ -1301,7 +1301,9 @@ def grant_details(request, grant_id, grant_slug):
                     [f'{grant.grant_type.name} {title} Summary Last 60 Days', get_grant_sybil_profile(None, 60 * 24, grant.grant_type, index_on=item)],
                     [f'All {title} Summary Last 60 Days', get_grant_sybil_profile(None, 60 * 24, None, index_on=item)],
                 ]
-        _contributions = Contribution.objects.filter(subscription__grant=grant, subscription__is_postive_vote=True).prefetch_related('subscription', 'subscription__contributor_profile')
+        _contributions = Contribution.objects.none()
+        if request.GET.get('load_contributions'):
```

Comment:
open to a less janky way of doing this

Reply:
Looks fine to me, theres some other jank in here thats a bit inefficient but unrelated to your change here